x/bank/handler.go

```diff
@@ -24,13 +25,14 @@ func handleMsgSend(ctx sdk.Context, k Keeper, msg MsgSend) sdk.Result {
 	if !k.GetSendEnabled(ctx) {
 		return ErrSendDisabled(k.Codespace()).Result()
 	}
-	tags, err := k.SendCoins(ctx, msg.FromAddress, msg.ToAddress, msg.Amount)
+	resTags, err := k.SendCoins(ctx, msg.FromAddress, msg.ToAddress, msg.Amount)
```

Comment:
Let's be consistent -- let's use `tags`.

Reply:
`tags` conflicts with the pkg name, so we should either rename the import the package with an alias (e.g `banktags "github.com/cosmos/cosmos-sdk/x/bank/tags"`) or stick with `resTags`. Your call

Reply:
Ahhh crap. Ok, `resTags` is fine 👍